src/Lut.ts

```diff
@@ -464,7 +466,8 @@ export function remapControlPoints(
 
   // remap all cp x values.
   // interpolate all new colors and opacities
-  // then see if we need to clip?
+  // Do not clip values outside of 0-255. This is important to
```

Comment:
I've noticed that the removed code addressed control points outside the 0-255 range, and it appears we're moving away from clipping these values. Can you share more about this change?

Reply:
The control points array is used to generate a densely sampled lookup table.  The lookup table is in the form of an array of result values.  The input values to the lookup table can not go outside of the 0-255 domain (strict array bounds).  The control points are just defining vertices of a piecewise linear function, and the lookup table is also representing the same function.   

If we clip the control points to the 0-255 domain when remapping them, then we lose information when the remapping would extend outside the bounds.  This is because they might be remapped AGAIN and return within bounds again.    It's ok for the control point values to be out of bounds as long as we build the lookup table array correctly.    I can explain this more in person if you'd like.

Reply:
Thank you for taking the time to explain how the control points and lookup table interact regarding the bounds, it makes so much sense!